Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/nbclient/client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/asyncio/base_events.py", line 691, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from sympde.expr import BilinearForm, LinearForm, integral
from sympde.expr     import find, EssentialBC, Norm, SemiNorm
from sympde.topology import VectorFunctionSpace, Cube, element_of
from sympde.calculus import grad, dot, inner, outer, cross, div
from sympde.core import Constant
from sympde.core import Matrix, Vector, Transpose

domain = Cube()

I = Matrix([[1, 0, 0], [0, 1, 0], [0, 0, 1]], name='I')

kappa = Constant('kappa', is_real=True)
mu    = Constant('mu',    is_real=True)
rho   = Constant('rho',   is_real=True)

epsilon = lambda w: 0.5*(grad(w) + Transpose(grad(w)))
sigma   = lambda w: kappa * div(w) * I + 2 * mu * epsilon(w)

V = VectorFunctionSpace('V', domain)

x,y,z = domain.coordinates

u,v = [element_of(V, name=i) for i in ['u', 'v']]
------------------


[31m---------------------------------------------------------------------------[39m
[31mImportError[39m                               Traceback (most recent call last)
[36mCell[39m[36m [39m[32mIn[1][39m[32m, line 6[39m
[32m      4[39m [38;5;28;01mfrom[39;00m[38;5;250m [39m[34;01msympde[39;00m[34;01m.[39;00m[34;01mcalculus[39;00m[38;5;250m [39m[38;5;28;01mimport[39;00m grad, dot, inner, outer, cross, div
[32m      5[39m [38;5;28;01mfrom[39;00m[38;5;250m [39m[34;01msympde[39;00m[34;01m.[39;00m[34;01mcore[39;00m[38;5;250m [39m[38;5;28;01mimport[39;00m Constant
[32m----> [39m[32m6[39m [38;5;28;01mfrom[39;00m[38;5;250m [39m[34;01msympde[39;00m[34;01m.[39;00m[34;01mcore[39;00m[38;5;250m [39m[38;5;28;01mimport[39;00m Matrix, Vector, Transpose
[32m      8[39m domain = Cube()
[32m     10[39m I = Matrix([[[32m1[39m, [32m0[39m, [32m0[39m], [[32m0[39m, [32m1[39m, [32m0[39m], [[32m0[39m, [32m0[39m, [32m1[39m]], name=[33m'[39m[33mI[39m[33m'[39m)

[31mImportError[39m: cannot import name 'Vector' from 'sympde.core' (/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/sympde/core/__init__.py)

